src/ChooseArea.svelte

```diff
@@ -72,9 +73,9 @@
       });
 
       map.on("mousemove", layer, (e) => {
-        if (e.features.length > 0) {
+        if (e.features!.length > 0) {
```

Comment:
Maybe we should use error handling, like optional chaining with `?`, more? But I've never seen when these events fire without features